src/App.tsx

```diff
@@ -179,6 +197,63 @@ export const App = ({ shortUrl, user }: Props) => {
 
   const showPagination = totalPages > 1;
 
+  if (isPreview) {
+    return (
+      <div className={commentContainerStyles}>
+        {user && (
+          <CommentForm
+            shortUrl={shortUrl}
+            onAddComment={onAddComment}
+            user={user}
+          />
+        )}
+        <div className={picksWrapper}>
+          {picks && picks.length ? (
+            <>
+              {picks.slice(0, 2).map(pick => (
+                <TopPick comment={pick} />
+              ))}
+            </>
+          ) : (
+            <>
+              {comments.slice(0, 2).map(comment => (
+                <CommentContainer
+                  key={comment.id}
+                  comment={comment}
+                  pillar="news"
+                  shortUrl={shortUrl}
+                  onAddComment={onAddComment}
+                  user={user}
+                  threads={filters.threads}
+                  commentBeingRepliedTo={commentBeingRepliedTo}
+                  setCommentBeingRepliedTo={setCommentBeingRepliedTo}
+                />
```

Comment:
good catch, its hard to tell looking at the frontend code. I assumed yes in this case, but maybe this is something we dnt wish to do?

Reply:
Turns out we do show replies in the preview. And it is long!

I'd still be keen to get a steer from Rob/the mods because just because FE does something, doesn't make it right!

![Screenshot 2020-03-06 at 13 32 28](https://user-images.githubusercontent.com/1336821/76088765-7f216800-5fb0-11ea-8aa5-a916b8840ba8.jpg)